src/Microsoft.Extensions.Configuration.AzureAppConfiguration/FeatureManagement/FeatureManagementConstants.cs

```diff
@@ -44,5 +44,10 @@ internal class FeatureManagementConstants
         public const string ETag = "ETag";
         public const string FeatureFlagId = "FeatureFlagId";
         public const string FeatureFlagReference = "FeatureFlagReference";
+
+        // Dotnet schema keys
```

Comment:
I believe those were all as a result of variants, so since the feature management library is only processing flags as .NET schema if they don't include variants, these fields shouldn't be necessary. @zhiyuanliang-ms can you fact check me here?

Reply:
The field `AlwaysOn` is still being used by .NET schema. [ref](https://github.com/microsoft/FeatureManagement-Dotnet/blob/main/src/Microsoft.FeatureManagement/ConfigurationFeatureDefinitionProvider.cs#L220)  [doc](https://learn.microsoft.com/en-us/azure/azure-app-configuration/feature-management-dotnet-reference?pivots=stable-version#feature-flag-declaration)

In our [preview release](https://github.com/microsoft/FeatureManagement-Dotnet/blob/preview/src/Microsoft.FeatureManagement/FeatureManager.cs#L483), we even support `On` filter which was introduced by @Amer's [PR](https://github.com/microsoft/FeatureManagement-Dotnet/pull/250/files#diff-7726d09bdcf1556c447fe12c6bd0f83385916697784034cec5d959edf8e7021aR214-R215) to support variant.

A feature flag like this:
``` json
{
  "id": "AlwaysOnFeature",
  "enabled": true
}
```
will look like this in .NET schema
``` json
"AlwaysOnFeature": {
  "EnabledFor": [
     {"Name": "AlwaysOn"}
  ]
}
```

Reply:
But .NET schema also support declaring feature flag in this way:
``` json
"AlwaysOnFeature": true
```
That's how currently .NET provider does. [ref](https://github.com/Azure/AppConfiguration-DotnetProvider/blob/main/src/Microsoft.Extensions.Configuration.AzureAppConfiguration/FeatureManagement/FeatureManagementKeyValueAdapter.cs#L35)